File: src/dataset/go/two_fer_Exercism_go_thinkverse/gemini/GeminiFlash_two_fer_v3.go
package twofer

func ShareWith(name string) string {
	if name == "" {
		name = "you"
	}

	result := make([]byte, 0, 30+len(name))
	result = append(result, "One for "...)
	result = append(result, ' ')
	result = append(result, name...)
	result = append(result, ", one for me."...)

	return string(result)
}

File: src/dataset/typescript/two-fer_Exercism-typescript-thewanionly/gemini/GeminiFlash_two-fer_v3.js
"use strict";
/**
 * This stub is provided to make it straightforward to get started.
 */
Object.defineProperty(exports, "__esModule", { value: true });
exports.twoFer = twoFer;
function twoFer(name = 'you') {
    const recipient = name || 'you';
    return `One for ${recipient}, one for me.`;
}
